### DISCUSSION TITLE
The chloroplast genome assembly is not a circle structure

### DISCUSSION THREAD

**Discussion by xhx-1** [Kinggerm/GetOrganelle#364] [Q&A]
Hello, thank you very much for the software you have developed to help me in my studies. I used getorganelle assembly, changed k, R parameters and even used --reduce-reads-for-coverage, but it didn't loop. What should I do to solve this problem please?

**Comment by xhx-1** [comment]
[files.zip](https://github.com/user-attachments/files/17043074/files.zip)

**Comment by huidejiandan** [comment]
你好，解决了吗，我也遇到这个问题了

**Comment by huidejiandan** [comment]
> 您好，非常感谢您开发的软件来帮助我的学习。我使用了 getorganelle assembly，更改了 k、R 参数，甚至使用了 --reduce-reads-for-coverage，但它没有循环。请问我该怎么办来解决这个问题？

你好解决了吗，我也遇到这个问题了

**Comment by xhx-1** [comment]
抱歉抱歉，这个问题没有解决




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Kinggerm/GetOrganelle"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年1月4日(星期六) 晚上9:19
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Kinggerm/GetOrganelle] The chloroplast genome assembly is not a circle structure (Issue #349)





  
您好，非常感谢您开发的软件来帮助我的学习。我使用了 getorganelle assembly，更改了 k、R 参数，甚至使用了 --reduce-reads-for-coverage，但它没有循环。请问我该怎么办来解决这个问题？
  
你好解决了吗，我也遇到这个问题了

**Comment by JianjunJin** [comment]
1. 发issue必须遵守模板，否则可能会被忽略或者被转移到Discussion，多次不遵守会被直接删帖；
2. 不成环不属于issue，请直接在Discussion求助；
3. 不成环的时候附带图形文件必须是`assembly_graph.fastg.extend-*.fastg`，而非`embplant_pt.*.selected_graph.gfa`
